documentation/docs/pages/operators/community-counsel/isp-list.mdx

```diff
@@ -14,12 +12,10 @@ If you came across any legal findings, please share them in our [list of jurisdi
 
 While we trust that Nym node operators are honest, we would like to ask everyone to do your own research.
 
-```admonish caution title=""
-To edit or add information to the ISP list, make changes to the csv file located [here](https://github.com/nymtech/nym/blob/develop/documentation/operators/src/data/isp-sheet.csv) and submit your edits as a pull request according to [this guide](add-content.md).
-```
+<Callout color="danger" emoji="">
+To edit or add information to the ISP list, make changes to the csv file located [here](https://github.com/nymtech/nym/blob/develop/documentation/docs/data/csv/isp-sheet.csv) and submit your edits as a pull request according to [this guide](add-content.mdx).
+</Callout>
 
-```admonish note title=""
-As of now the list is quite short. When it grows, we can divide it according the localities of the listed ISPs.
 ```
-
-<!--cmdrun python3 ../../../scripts/csv2md.py ../data/isp-sheet.csv -s 0 -->
+<!-- cmdrun python3 ../../../scripts/csv2md.py ../data/isp-sheet.csv -s 0 -->
```

Comment:
remove for now?

Reply:
I left all cmdrun for easier location post merge